providers/os/provider/provider.go

```diff
@@ -89,7 +89,7 @@ func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error)
 		port = 5985
 	case "vagrant":
 		conf.Type = "vagrant"
-	case "docker":
+	case "docker", "container":
```

Comment:
i think we should keep those separate. there should be a difference between  `container image scan` which should attempt to pull from the registry you've specified and `docker image scan` which should try to pull from the local docker that is running

Reply:
But on this switch statement, bot do exactly the same thing on `image` as far as I can tell (and also for all other options)

Reply:
I agree with Preslav. This is how it should work

Reply:
Okay, we can close the issue as intended behavior then?

Reply:
No because `cnspec scan container docker.io/pmuench/dvwa-container-escape` needs to work without docker